simplq/src/components/pages/Admin/ControlScroll.js

```diff
@@ -0,0 +1,44 @@
+// left: 37, up: 38, right: 39, down: 40,
```

Comment:
Can you rather create a `simplq/src/utils` folder and keep there there?

Reply:
Since we already have the `/common` folder, adding another `/utils` folder seems like overkill to me. And since this is not really used anywhere else, let's just keep it here for now @daltonfury42 ?

Reply:
In my mind `simplq/src/components/common` is for reusable react components. I was thinking `simplq/src/util` will be for non-component (pure js) utility functions like these. In that sense, our current [utilFns.js](https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/common/utilFns.js) also is looking a bit odd. 

It's generally a good idea to separate out utils functions into common folders, else I've seen that they end up getting redefined than reused over time.

Reply:
Fair enough, we can make the folder and split up utilFns into meaningful modules. But we should keep a rule to limit stuff there to constants or functions that are/will be used atleast twice in different components, or it's just unnecessary overhead imo. In this ControlScroll.js file, I don't feel like anything has the potential to be reused anywhere else, atleast not in it's current state.

Reply:
Alright.